tests/ui/offset-of/offset-of-builtin.stderr

```diff
@@ -4,62 +4,43 @@ error: expected one of `!`, `(`, `+`, `,`, `::`, or `<`, found `)`
 LL |     builtin # offset_of(NotEnoughArguments);
    |                                           ^ expected one of `!`, `(`, `+`, `,`, `::`, or `<`
 
-error: expected identifier, found `,`
-  --> $DIR/offset-of-builtin.rs:16:41
+error: expected expression, found `)`
```

Comment:
this is a *slight* regression from the state before, but I think it's totally okay to have it, given that it's in builtin syntax only.